adapters/conversant/conversant.go

```diff
@@ -130,6 +139,18 @@ func parseCnvrParams(imp *openrtb2.Imp, cnvrExt openrtb_ext.ExtImpConversant) {
 			imp.Video.MaxDuration = *cnvrExt.MaxDuration
 		}
 	}
+	if imp.BidFloor > 0 && imp.BidFloorCur != "" && strings.ToUpper(imp.BidFloorCur) != "USD" {
+		floor, err := reqInfo.ConvertCurrency(imp.BidFloor, imp.BidFloorCur, "USD")
+		if err != nil {
+			return []error{&errortypes.BadInput{
+				Message: fmt.Sprintf("Unable to convert provided bid floor currency from %s to USD", imp.BidFloorCur),
+			}}
+		} else if floor > 0 {
+			imp.BidFloorCur = "USD"
+			imp.BidFloor = floor
+		}
```

Comment:
Why would the floor not be greater than 0 after a currency conversion?

Reply:
I was thinking I didn't want a negative floor, but now I think better to pass it through than mysteriously dropping it.  fixed